src/containers/GameListPage.js

```diff
@@ -2,10 +2,13 @@ import { connect } from 'react-redux';
 import GameListPage from '../components/GameListPage';
 import { getRooms } from '../actions/getRooms';
```

Comment:
https://github.com/duyoji/oni-on-cheese/pull/92/files#diff-7c7034399b0fcdfcf2b08293bf0aaf4eL9

According to modification of GameListPage component, I think you no longer use getRooms function. 
If so, please delete related code about getRooms from component, container.